suite-native/transactions/src/components/TransactionDetail/TransactionDetailValuesSheet.tsx

```diff
@@ -0,0 +1,128 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+
+import { selectCoins } from '@suite-common/wallet-core';
+import { useFormatters } from '@suite-common/formatters';
+import { WalletAccountTransaction } from '@suite-common/wallet-types';
+import { Card, Table, Td, Text, Th, Tr, VStack } from '@suite-native/atoms';
+import { formatNetworkAmount, toFiatCurrency } from '@suite-common/wallet-utils';
+import { FiatCurrency } from '@suite-common/suite-config';
+
+import { TransactionDetailSheet } from './TransactionDetailSheet';
+
+type TransactionDetailValuesSheetProps = {
+    isVisible: boolean;
+    transaction: WalletAccountTransaction;
+    fiatCurrency: FiatCurrency;
+    onSheetVisibilityChange: () => void;
+};
+
+/**
+ * Creates table header cell which contains percentual difference of transaction profit/loss.
+ */
+const renderTodayHeaderCell = (historicalPrice: string | number, actualPrice: string | number) => {
```

Comment:
This could be converted to normal component instead of function

Reply:
fixed in 595577f78